perun-openapi/openapi.yml

```diff
@@ -14235,6 +14251,34 @@ paths:
         default:
           $ref: '#/components/responses/ExceptionResponse'
 
+  /json/resourcesManager/getEnrichedBansForResource:
+    get:
+      tags:
+        - ResourcesManager
+      operationId: getEnrichedBansForResource
+      summary: Get all enriched bans for members on the resource.
+      parameters:
+        - $ref: '#/components/parameters/resourceId'
```

Comment:
Shouldn't we also add the attrNames parameter in both of the methods ? And also mention them in the javadoc in `ResourcesManagerMethod`.